model.py

```diff
@@ -0,0 +1,82 @@
+from matplotlib.pylab import f
+import numpy as np
+import pandas as pd
+import matplotlib as plib
+import torch
+from torch import nn
+
+class Generator(nn.Module):
+    def __init__(self, color_dim, out_dim, device):
+        self.device = device
+        self.kernel_size = 4
+        super().__init__()
+        ngf = 32
+        #self.kernel = nn.Parametar(torch.randn(broj_slova, z_dim))
+        self.gen = nn.Sequential(            
+            nn.ConvTranspose2d(color_dim * 100, ngf * 8, self.kernel_size, 1, 0, bias=False, device=self.device),
+            nn.BatchNorm2d(ngf * 8),
+            nn.ReLU(True),
+            
+            nn.ConvTranspose2d(ngf * 8, ngf * 4, self.kernel_size, 2, 1, bias=False, device=self.device),
+            nn.BatchNorm2d(ngf * 4),
+            nn.ReLU(True),
+            
+            nn.ConvTranspose2d( ngf * 4, ngf * 2, self.kernel_size, 2, 1, bias=False, device=self.device),
+            nn.BatchNorm2d(ngf * 2),
+            nn.ReLU(True),
+            
+            nn.ConvTranspose2d( ngf * 2, ngf, self.kernel_size, 2, 1, bias=False, device=self.device),
+            nn.BatchNorm2d(ngf),
+            nn.ReLU(True),
+            
+            nn.ConvTranspose2d( ngf, out_dim, self.kernel_size, 2, 1, bias=False ,device=self.device),
+            nn.Tanh()
+        ).to(self.device)
```

Comment:
i ovaj to.(device)

Reply:
to mi milomir reko za svaki sluc da staavimo jer nece graficku da inicijalizuje